*** ./smali/android/provider/Settings$System.smali	2016-03-24 11:21:54.715329364 +0100
--- /home/dadi11/miui/g3/temp/new_smali/framework.jar.out/./smali/android/provider/Settings$System.smali	2016-03-24 11:19:05.087329364 +0100
***************
*** 1527,1532 ****
--- 1527,1534 ----
      iput v0, p1, Landroid/content/res/Configuration;->fontScale:F
  
      :cond_0
+     invoke-static {p0, p1, p2}, Landroid/provider/MiuiSettings;->getConfigurationForUser(Landroid/content/ContentResolver;Landroid/content/res/Configuration;I)V
+ 
      return-void
  .end method
  
