*** ./smali/com/android/server/power/PowerManagerService$WakeLock.smali	2016-03-23 22:37:13.315329364 +0100
--- /home/dadi11/miui/g3/temp/new_smali/services.jar.out/./smali/com/android/server/power/PowerManagerService$WakeLock.smali	2016-03-23 22:34:08.271329364 +0100
***************
*** 12,23 ****
  .end annotation
  
  .annotation system Ldalvik/annotation/InnerClass;
!     accessFlags = 0x12
      name = "WakeLock"
  .end annotation
  
  
  # instance fields
  .field public mFlags:I
  
  .field public final mLock:Landroid/os/IBinder;
--- 12,25 ----
  .end annotation
  
  .annotation system Ldalvik/annotation/InnerClass;
!     accessFlags = 0x10
      name = "WakeLock"
  .end annotation
  
  
  # instance fields
+ .field public mDisabled:Z
+ 
  .field public mFlags:I
  
  .field public final mLock:Landroid/os/IBinder;
***************
*** 309,314 ****
--- 311,327 ----
  
      move-result-object v0
  
+     iget-boolean v1, p0, Lcom/android/server/power/PowerManagerService$WakeLock;->mDisabled:Z
+ 
+     if-eqz v1, :cond_miui_0
+ 
+     const-string v1, " DISABLED"
+ 
+     :goto_miui_0
+     invoke-virtual {v0, v1}, Ljava/lang/StringBuilder;->append(Ljava/lang/String;)Ljava/lang/StringBuilder;
+ 
+     move-result-object v0
+ 
      const-string v1, " (uid="
  
      invoke-virtual {v0, v1}, Ljava/lang/StringBuilder;->append(Ljava/lang/String;)Ljava/lang/StringBuilder;
***************
*** 356,361 ****
--- 369,379 ----
      move-result-object v0
  
      return-object v0
+ 
+     :cond_miui_0
+     const-string v1, ""
+ 
+     goto :goto_miui_0
  .end method
  
  .method public updateProperties(ILjava/lang/String;Ljava/lang/String;Landroid/os/WorkSource;II)V
