*** ./smali/com/android/internal/telephony/uicc/IccRecords.smali	2016-03-24 11:21:26.031329364 +0100
--- /home/dadi11/miui/g3/temp/new_smali/telephony-common.jar.out/./smali/com/android/internal/telephony/uicc/IccRecords.smali	2016-03-24 11:18:38.711329364 +0100
***************
*** 63,68 ****
--- 63,70 ----
  
  .field protected mMailboxIndex:I
  
+ .field protected mMiuiAdnCache:Lcom/android/internal/telephony/uicc/MiuiAdnRecordCache;
+ 
  .field protected mMncLength:I
  
  .field protected mMsisdn:Ljava/lang/String;
***************
*** 1149,1154 ****
--- 1151,1165 ----
      return-object v0
  .end method
  
+ .method public getMiuiAdnCache()Lcom/android/internal/telephony/uicc/MiuiAdnRecordCache;
+     .locals 1
+ 
+     .prologue
+     iget-object v0, p0, Lcom/android/internal/telephony/uicc/IccRecords;->mMiuiAdnCache:Lcom/android/internal/telephony/uicc/MiuiAdnRecordCache;
+ 
+     return-object v0
+ .end method
+ 
  .method public getMsisdnAlphaTag()Ljava/lang/String;
      .locals 1
  
***************
*** 1449,1454 ****
--- 1460,1469 ----
  
      invoke-virtual {v0}, Lcom/android/internal/telephony/uicc/AdnRecordCache;->reset()V
  
+     iget-object v0, p0, Lcom/android/internal/telephony/uicc/IccRecords;->mMiuiAdnCache:Lcom/android/internal/telephony/uicc/MiuiAdnRecordCache;
+ 
+     invoke-virtual {v0}, Lcom/android/internal/telephony/uicc/MiuiAdnRecordCache;->reset()V
+ 
      iget-object v0, p0, Lcom/android/internal/telephony/uicc/IccRecords;->mParentApp:Lcom/android/internal/telephony/uicc/UiccCardApplication;
  
      invoke-virtual {v0}, Lcom/android/internal/telephony/uicc/UiccCardApplication;->getState()Lcom/android/internal/telephony/uicc/IccCardApplicationStatus$AppState;
***************
*** 2126,2128 ****
--- 2141,2178 ----
  
      return-void
  .end method
+ 
+ .method protected setSystemProperty(Ljava/lang/String;Ljava/lang/String;)V
+     .locals 2
+     .param p1, "property"    # Ljava/lang/String;
+     .param p2, "value"    # Ljava/lang/String;
+ 
+     .prologue
+     invoke-static {}, Lcom/android/internal/telephony/PhoneFactory;->getDefaultPhone()Lcom/android/internal/telephony/Phone;
+ 
+     move-result-object v0
+ 
+     .local v0, "phone":Lcom/android/internal/telephony/Phone;
+     :goto_0
+     instance-of v1, v0, Lcom/android/internal/telephony/PhoneProxy;
+ 
+     if-eqz v1, :cond_0
+ 
+     check-cast v0, Lcom/android/internal/telephony/PhoneProxy;
+ 
+     .end local v0    # "phone":Lcom/android/internal/telephony/Phone;
+     invoke-virtual {v0}, Lcom/android/internal/telephony/PhoneProxy;->getActivePhone()Lcom/android/internal/telephony/Phone;
+ 
+     move-result-object v0
+ 
+     .restart local v0    # "phone":Lcom/android/internal/telephony/Phone;
+     goto :goto_0
+ 
+     :cond_0
+     check-cast v0, Lcom/android/internal/telephony/PhoneBase;
+ 
+     .end local v0    # "phone":Lcom/android/internal/telephony/Phone;
+     invoke-virtual {v0, p1, p2}, Lcom/android/internal/telephony/PhoneBase;->setSystemProperty(Ljava/lang/String;Ljava/lang/String;)V
+ 
+     return-void
+ .end method
\ No newline at end of file
