*** ./smali/com/android/internal/telephony/uicc/RuimRecords$EfCsimSpnLoaded.smali	2016-03-23 22:37:00.255329364 +0100
--- /home/dadi11/miui/g3/temp/new_smali/telephony-common.jar.out/./smali/com/android/internal/telephony/uicc/RuimRecords$EfCsimSpnLoaded.smali	2016-03-23 22:33:55.347329364 +0100
***************
*** 263,268 ****
--- 263,272 ----
  
      invoke-static {v7, v8}, Landroid/os/SystemProperties;->set(Ljava/lang/String;Ljava/lang/String;)V
  
+     iget-object v9, p0, Lcom/android/internal/telephony/uicc/RuimRecords$EfCsimSpnLoaded;->this$0:Lcom/android/internal/telephony/uicc/RuimRecords;
+ 
+     invoke-virtual {v9, v7, v8}, Lcom/android/internal/telephony/uicc/RuimRecords;->setSystemProperty(Ljava/lang/String;Ljava/lang/String;)V
+ 
      goto :goto_2
  
      :pswitch_1
