*** ./smali/com/android/internal/telephony/ITelephonyRegistry$Stub.smali	2016-03-24 11:21:29.299329364 +0100
--- /home/dadi11/miui/g3/temp/new_smali/framework2.jar.out/./smali/com/android/internal/telephony/ITelephonyRegistry$Stub.smali	2016-03-24 11:18:42.179329364 +0100
***************
*** 607,613 ****
  
      goto/16 :goto_0
  
!     nop
  
      :sswitch_data_0
      .sparse-switch
--- 607,663 ----
  
      goto/16 :goto_0
  
!     :sswitch_fe
!     const-string v0, "com.android.internal.telephony.ITelephonyRegistry"
! 
!     invoke-virtual {p2, v0}, Landroid/os/Parcel;->enforceInterface(Ljava/lang/String;)V
! 
!     invoke-virtual {p2}, Landroid/os/Parcel;->readStrongBinder()Landroid/os/IBinder;
! 
!     move-result-object v0
! 
!     invoke-static {v0}, Lmiui/telephony/IMiuiTelephony$Stub;->asInterface(Landroid/os/IBinder;)Lmiui/telephony/IMiuiTelephony;
! 
!     move-result-object v1
! 
!     .local v1, "_arg0":Lmiui/telephony/IMiuiTelephony;
!     invoke-virtual {p0, v1}, Lcom/android/internal/telephony/ITelephonyRegistry$Stub;->setMiuiTelephony(Lmiui/telephony/IMiuiTelephony;)V
! 
!     invoke-virtual {p3}, Landroid/os/Parcel;->writeNoException()V
! 
!     const/4 v0, 0x1
! 
!     goto/16 :goto_0
! 
!     :sswitch_ff
!     const-string v0, "com.android.internal.telephony.ITelephonyRegistry"
! 
!     invoke-virtual {p2, v0}, Landroid/os/Parcel;->enforceInterface(Ljava/lang/String;)V
! 
!     invoke-virtual {p0}, Lcom/android/internal/telephony/ITelephonyRegistry$Stub;->getMiuiTelephony()Lmiui/telephony/IMiuiTelephony;
! 
!     move-result-object v11
! 
!     .local v11, "_result":Lmiui/telephony/IMiuiTelephony;
!     invoke-virtual {p3}, Landroid/os/Parcel;->writeNoException()V
! 
!     if-eqz v11, :cond_ff
! 
!     invoke-interface {v11}, Lmiui/telephony/IMiuiTelephony;->asBinder()Landroid/os/IBinder;
! 
!     move-result-object v0
! 
!     :goto_ff
!     invoke-virtual {p3, v0}, Landroid/os/Parcel;->writeStrongBinder(Landroid/os/IBinder;)V
! 
!     const/4 v0, 0x1
! 
!     goto/16 :goto_0
! 
!     :cond_ff
!     const/4 v0, 0x0
! 
!     goto :goto_ff
  
      :sswitch_data_0
      .sparse-switch
***************
*** 623,628 ****
--- 673,680 ----
          0xa -> :sswitch_a
          0xb -> :sswitch_b
          0xc -> :sswitch_c
+         0xfe -> :sswitch_fe
+         0xff -> :sswitch_ff
          0x5f4e5446 -> :sswitch_0
      .end sparse-switch
  .end method
