*** ./smali/android/media/MediaMetadataEditor.smali	2016-03-23 22:37:37.659329364 +0100
--- /home/dadi11/miui/g3/temp/new_smali/framework.jar.out/./smali/android/media/MediaMetadataEditor.smali	2016-03-23 22:34:33.279329364 +0100
***************
*** 140,145 ****
--- 140,151 ----
  
      sget-object v0, Landroid/media/MediaMetadataEditor;->METADATA_KEYS_TYPE:Landroid/util/SparseIntArray;
  
+     const/16 v1, 0x3e8
+ 
+     invoke-virtual {v0, v1, v2}, Landroid/util/SparseIntArray;->put(II)V
+ 
+     sget-object v0, Landroid/media/MediaMetadataEditor;->METADATA_KEYS_TYPE:Landroid/util/SparseIntArray;
+ 
      const/16 v1, 0x64
  
      invoke-virtual {v0, v1, v5}, Landroid/util/SparseIntArray;->put(II)V
