*** ./smali/com/android/internal/telephony/PhoneProxy.smali	2016-03-24 11:21:25.783329364 +0100
--- /home/dadi11/miui/g3/temp/new_smali/telephony-common.jar.out/./smali/com/android/internal/telephony/PhoneProxy.smali	2016-03-24 11:18:38.463329364 +0100
***************
*** 1547,1552 ****
--- 1547,1565 ----
      return v0
  .end method
  
+ .method public getMiuiIccPhoneBookInterfaceManager()Lcom/android/internal/telephony/MiuiIccPhoneBookInterfaceManager;
+     .locals 1
+ 
+     .prologue
+     iget-object v0, p0, Lcom/android/internal/telephony/PhoneProxy;->mActivePhone:Lcom/android/internal/telephony/Phone;
+ 
+     invoke-interface {v0}, Lcom/android/internal/telephony/Phone;->getMiuiIccPhoneBookInterfaceManager()Lcom/android/internal/telephony/MiuiIccPhoneBookInterfaceManager;
+ 
+     move-result-object v0
+ 
+     return-object v0
+ .end method
+ 
  .method public getMsisdn()Ljava/lang/String;
      .locals 1
  
