*** ./smali/android/widget/Toast$TN.smali	2016-03-23 22:37:21.927329364 +0100
--- /home/dadi11/miui/g3/temp/new_smali/framework.jar.out/./smali/android/widget/Toast$TN.smali	2016-03-23 22:34:16.563329364 +0100
***************
*** 168,173 ****
--- 168,182 ----
  
  
  # virtual methods
+ .method getParams()Landroid/view/WindowManager$LayoutParams;
+     .locals 1
+ 
+     .prologue
+     iget-object v0, p0, Landroid/widget/Toast$TN;->mParams:Landroid/view/WindowManager$LayoutParams;
+ 
+     return-object v0
+ .end method
+ 
  .method public handleHide()V
      .locals 2
  
