*** ./smali/android/webkit/WebView.smali	2016-03-23 22:37:15.751329364 +0100
--- /home/dadi11/miui/g3/temp/new_smali/framework.jar.out/./smali/android/webkit/WebView.smali	2016-03-23 22:34:10.635329364 +0100
***************
*** 167,172 ****
--- 167,176 ----
  
      invoke-static {}, Landroid/webkit/CookieSyncManager;->setGetInstanceIsAllowed()V
  
+     invoke-static {p0, p1}, Landroid/webkit/WebViewInjector;->initEgretLoader(Landroid/webkit/WebView;Landroid/content/Context;)V
+ 
+     invoke-static {p1}, Lmiui/content/res/ThemeFontChangeHelper;->markWebViewCreated(Landroid/content/Context;)V
+ 
      return-void
  
      :cond_1
